classes/components/forms/site/PKPSiteStatisticsForm.inc.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * @file classes/components/forms/site/PKPSiteStatisticsForm.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPSiteStatisticsForm
+ * @ingroup classes_controllers_form
+ *
+ * @brief A preset form for the site statistics settings.
+ */
+
+namespace PKP\components\forms\site;
+
+use PKP\components\forms\FieldOptions;
+use PKP\components\forms\FieldText;
+use PKP\components\forms\FormComponent;
+
+define('FORM_SITE_STATISTICS', 'siteStatistics');
+
+class PKPSiteStatisticsForm extends FormComponent
+{
+    /** @copydoc FormComponent::$id */
+    public $id = FORM_SITE_STATISTICS;
+
+    /** @copydoc FormComponent::$method */
+    public $method = 'PUT';
+
+    /**
+     * Constructor
+     *
+     * @param string $action URL to submit the form to
+     * @param array $locales Supported locales
+     * @param Site $site
+     */
+    public function __construct(string $action, array $locales, \PKP\site\Site $site)
+    {
+        $this->action = $action;
+        $this->locales = $locales;
+
+        $this->addGroup([
+            'id' => 'archive',
+        ])
+            ->addField(new FieldOptions('archivedUsageStatsLogFiles', [
+                'label' => __('manager.settings.statistics.archivedUsageStatsLogFiles'),
+                'description' => __('manager.settings.statistics.archivedUsageStatsLogFiles.description'),
```

Comment:
Oh, good catch -- I think I copied that key from the old usageStats plugin, but now we do not need that configuration any more because we use gzopen... I will change the text...